stdlib/asm/crypto/stark/utils.masm

```diff
@@ -1,5 +1,7 @@
 const.ROOT_UNITY=7277203076849721926
 const.SEED_PTR=4294967293
+const.SEED_PTR_2=4294967294
+const.SEED_PTR_1=4294967295
```

Comment:
Similar as the other comment re `SEED_PTR_0`. Also, maybe we should have these just in `random_coin.masm` and provide relevant functions from there to access them (e.g., something like `get_seed` and `set_seed` to load seed onto the stack and save it to memory, respectively).

Reply:
I have added get and set procedures for each of the three components of the random coin. Having one get and one set method makes things a bit more cumbersome sometimes when we only need the first half of the rate portion of the state.